### ISSUE TITLE
Custom Date string resets to default on each boot / panel reload

### ISSUE THREAD

**Issue by HolarMusic** [danielpinto8zz6/budgie-calendar-applet#12]
###   Steps to reproduce
- Change the "Custom Date" string
- Reload your panel

**Comment by zacharyPuthoff** [comment]
I'll second this as an issue --- occurs on panel reboot or system restart. 

Running Solus Version 3.26.2, budgie-desktop 10.4, fresh install, all installed software from repositories.

**Comment by danielpinto8zz6** [comment]
I'll fix this soon as possible, I'm little busy right now, but I'll try to fix this this week

**Comment by danielpinto8zz6** [comment]
Ok guys, it's fixed, https://github.com/danielpinto8zz6/budgie-calendar-applet/commit/22f7026fbc5f5bbf5576637c7c348556178bc0d2
I'll not release now, I'll make some updates, including new translations

**Comment by zacharyPuthoff** [comment]
Excellent! Thanks for the fast fix @danielpinto8zz6!

**Comment by dreamcat4** [comment]
Hi. I encountered this problem, from the ubuntu backports just a couple of days ago.

It seemed the setting was never written / saved out to the dconf database. The way I solved it was to write to the dconf database manually. By opening dconf editor and navigating to:

`com/github/danielpinto8zz6/budgie-calendar-applet/custom-format`

And then pasting my custom value for the time / date format string into the box there.


Your fix does not seem to have reached them yet. (the budgie backports ppa). My version is:

```
Package: budgie-calendar-applet
Version: 4.1-0ubuntu1~bionic
Maintainer: David Mohammed <fossfreedom@ubuntu.com>
APT-Sources: http://ppa.launchpad.net/ubuntubudgie/backports/ubuntu bionic/main amd64 Packages
```
